checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -72,7 +122,9 @@ public LockAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFlow) {
     protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
         return Collections.unmodifiableSet(
                 new HashSet<Class<? extends Annotation>>(
```

Comment:
fixed (I wish there were a good way to mark a comment as "fixed", for my own and others' benefit, without spamming everyone)